docs/src/starknet/account.md

```diff
@@ -0,0 +1,49 @@
+# Creating And Deploying Accounts
+
+Starknet Foundry cast supports entire account management flow with the `sncast account create` and `sncast account deploy` commands.
+
+Do the following to start interacting with the Starknet:
```

Comment:
I think it could use a rephrase. Maybe a sentence about that basically all interaction with starknet needs an account at some point, and that one could start off with an account creation? I also think it may be a good idea to say a few words about what is the difference between creating and deploying accounts.